core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -191,19 +191,44 @@
 #else
 #define NUM_L1_TABLES	1
 #endif
-uint64_t l1_xlation_table[NUM_L1_TABLES][CFG_TEE_CORE_NB_CORE][NUM_L1_ENTRIES]
+
+typedef uint64_t l1_xlat_tbl_t[NUM_L1_ENTRIES];
+typedef l1_xlat_tbl_t l1_xlat_tbls_t[CFG_TEE_CORE_NB_CORE];
```

Comment:
`l1_xlat_tbl_t` doesn't seem to be used other than here. I'd prefer typedefing a two dimensional array directly instead of using a helper type.